src/components/avatar/index.tsx

```diff
@@ -93,6 +124,20 @@ export type AvatarProps = Pick<AccessibilityProps, 'accessibilityLabel'> & {
    * fails (equiv. to Image.onError()).
    */
   onImageLoadError?: ImagePropsBase['onError'];
+  /**
+   * The name of the avatar user.
+   * Text initials will be generated from the name.
+   * Has lower priority than label.
+   */
+  name?: string;
+  /**
+   * Override the logic for creating initials of the user name (default from AvatarHelper)
+   */
+  getInitials?: (name?: string, limit?: number) => string;
```

Comment:
Avoid adding props that the user can have on their end. 
If the user wants to implement a custom logic for initials, they can do it and pass the `label` prop.

Reply:
But we want the coloring logic to only apply when `name` is sent, and `name` is ignored when `label` is sent.

Reply:
Removed